api/src/main/resources/application.yml

```diff
@@ -134,6 +134,8 @@ csm:
             namespace:
               name: RootManageSharedAccessKey
               key: null
+        # Due to PROD-8599, we rename eventhub's shared access policy env vars
+        usePasswordForAMQPConsumer: false
```

Comment:
For backward compatibility, I suggest you just append the new `CSM_AMQPCONSUMER_*` environment variables to the existing `AZURE_EVENT_HUB_SHARED_ACCESS_*` ones.
This extra `usePasswordForAMQPConsumer` property and its handling in the code are not needed, I think.